app/helpers/games_helper.rb

```diff
@@ -1,6 +1,9 @@
 module GamesHelper
   def render_piece(x, y)
-    piece = @game.pieces.find_by(x_position: x, y_position: y)
-    return "#{piece.type} #{piece.color} " if piece
+    if piece = @game.pieces.find_by(x_position: x, y_position: y)
+     return link_to "#{piece.type} #{piece.color}", piece_path
+    else
+     return "Move Too"
+   end 
```

Comment:
To make it more readable, in the controller, you could change the variable name to `@selected_piece`